clevertap-core/src/main/java/com/clevertap/android/sdk/Constants.java

```diff
@@ -229,6 +229,7 @@ public interface Constants {
     String KEY_ITEM_PROPERTIES = "itemProperties";
     String KEY_GEO_RADIUS_PROPERTIES = "geoRadius";
     String KEY_PROFILE_ATTR_NAME = "profileAttrName";
+    String KEY_FIRST_TIME_ONLY = "firstTimeOnly";
```

Comment:
Lets have constants belonging to feature/sub-feature in that particular file itself.

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/de3cc2d6bd83e3856f1ad0ea0b827ab92f7708a9